Dockerfile

```diff
@@ -0,0 +1,32 @@
+ARG BASE_SERVER_IMAGE=temporalio/base-server:1.2.0
+ARG GOPROXY
+
+##### UI builder #####
+FROM golang:1.17-alpine3.14 AS ui-builder
```

Comment:
Why not `base-builder`?

Reply:
i need `alpine3.14` because it has proper version of `npm`

I wanted to upgrade `base-builder` though according to the comment here it will break https://github.com/temporalio/temporal/pull/1874

Reply:
ok, you can leave this as temporary solution for now, but it won't work on our BuildKite and CICD because they both have docker 19.x. You will need to build this image manually.